translate/storage/xliff.py

```diff
@@ -526,7 +526,7 @@ def getlocations(self):
                 if sourceFile is not None and lineNumber is not None:
                     locations.append(sourceFile[1] + ":" + lineNumber[1])
 
-        return ", ".join(x for x in locations if x is not None)
+        return [x for x in locations if x is not None]
```

Comment:
I don't see where `None` would appear here. Seems you've just blindly copied this workaround from Weblate (where it is also no longer needed):

```suggestion
        return locations
```